docs/docs/sdn_controller.md

```diff
@@ -40,6 +40,68 @@ In the network creation view:
 - Only 1 encrypted GRE network and 1 encrypted VxLAN network per pool can exist at a time due to Open vSwitch limitation.
 :::
 
+### Network bonding
+
+#### Definition
+
+Network bonding, also known as link aggregation, involves combining two or more physical network interfaces into a single logical interface.
+
+The primary goals are to improve redundancy —so that a single cable or interface failure does not interrupt connectivity— and, depending on the bonding mode, to increase throughput by distributing traffic across multiple links.
+
+#### Supported bond types
+
+Xen Orchestra supports the following bond types:
+
+
+- **Active / Active (Balance-SLB)**  
+  - Does **not** require switch awareness (no LACP needed).  
+  - Balances traffic across all bond members.  
+  - Automatically shifts traffic to remaining members if one link fails.  
+  - May have compatibility issues with some low-end switches.
+
+- **Active / Backup**  
+  - Does **not** require switch awareness.  
+  - Uses one primary interface for traffic, switching to backup only upon link failure.
+
+- **LACP**  
+  - Requires the switch to support and be configured for LACP.  
+  - Offers rapid failover and can detect mid-span link failures even if the physical link stays up.
+  - Balances traffic across all bond members during normal operations.
+
+#### Creating a LACP bond
+
+1. Navigate to **New → Network** to open the **Create Network** page.  
+2. Choose the pool where you want the bond to be created.
+3. Enable the **Bonded network** toggle:
+    ![Bond creation interface](./assets/bond0.png)
+4. In the **Interface** dropdown, select two or more physical interfaces (PIFs) to include in the bond.  
+5. Provide a **name** and a **description** for the bond.
+6. Set the **Bond mode** to **LACP**.
+7. Leave the **MTU** field blank. This will work as-is in 99% of cases.
```

Comment:
`in most cases`, getting a number feels unlikely accurate as we have no idea of all use cases.